test/unit/lib/getPharmacies.js

```diff
@@ -13,18 +13,18 @@ const expect = chai.expect;
 describe('Nearby', () => {
   let geo = [];
   const searchPoint = { latitude: 53.797431921096, longitude: -1.55275457242333 };
-  const sillySearchPoint = { latitude: 49.9126167297363, longitude: -6.30890274047852 };
+  const remoteSearchPoint = { latitude: 49.9126167297363, longitude: -6.30890274047852 };
 
   describe('happy path', () => {
-    beforeEach('load data', () => {
+    before('load data', () => {
```

Comment:
What is the rational for switching from `beforeEach` to `before`. I assume it's based on cache being read-only so beforeEach is unnecessary?